Dockerfiles/agent/README.md

```diff
@@ -90,3 +90,32 @@ logs:
      sourcecategory: sourcecode
 ```
 For more examples of configuration files or agent capabilities (such as filtering, redacting, multiline, …) read [this documentation](https://docs.datadoghq.com/logs/#filter-logs).
+
+### Kubernetes
+
+To deploy the Agent in your Kubernetes cluster, you can use the manifest in manifests/agent.yaml.
+`kubectl create -f manifest/agent.yaml`
+Make sure you have the correct RBAC in place. You can use the files in manifest/rbac that contain the minimal requirements to collect events and perform the leader election.
+`kubectl create -f manifest/rbac`
+
+If you want the event collection to be resilient, please create a ConfigMap as follows:
+`kubectl create configmap configmapdatadogtoken --from-literal="event.tokenKey"="0"`
+See details in [Event Collection](#event-collection).
+
+### Event Collection
+<a name="event-collection"></a>
+Similarly to the Agent 5, the Agent 6 can collect events from the Kubernetes API server.
+First and foremost, you need to set the `collect_kubernetes_events` variable to true in the datadog.yaml, this can be achieved via the environment variable `DD_COLLECT_KUBERNETES_EVENTS` that is resolved at start time.
+You will need to allow the agent to be allowed to perform a few actions:
+
+- `get` and `update` of the `Configmaps` named `configmapdatadogtoken` to update and query the most up to date version token corresponding to the latest event stored in ETCD.
+- `list` and `watch` of the `Events` to pull the events from the API Server, format and submit them.
+
+
+The ConfigMap to store the `event.tokenKey` and the `event.tokenTimestamp` has to be deployed in the `default` namespace and be named `configmapdatadogtoken`
+One can simply run `kubectl create configmap configmapdatadogtoken --from-literal="event.tokenKey"="0"` .
+
+NB: you can set any resversion here, make sure it's **not** set to a value superior to the actual curent resversion.
```

Comment:
Not sure why we're talking about resource version here, isn' it more confusing than helping? Because I guess this shouldn't be modified no? (keep default behaviour)

Reply:
agreed, this should probably be internal details.